.github/workflows/build-images.yml

```diff
@@ -296,28 +252,26 @@ jobs:
           ref: ${{ needs.build-info.outputs.target-commit-sha }}
           persist-credentials: false
           submodules: recursive
-      - name: >
-          Checkout "main" branch to 'main-airflow' folder
-          to use ci/scripts from there.
+      - name: Checkout target branch to 'target-airflow' folder to use ci/scripts from there.
```

Comment:
(Don't have full context loaded in my head) Is there a reason we don't use the same action here?

Reply:
Yep. I noticed it when self-reviewed it and pushed an updated version just before :)